### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by z-samuel**
Hi, @edisonz0718 , this is the update for the README file. Please kindly review it, thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,140 @@
...
+```
+yarn add @story-protocol/core-sdk
+```
+
+### Setup the `.env` file for your project
+
+The second steps is to create a file named `.env` in the root directory of the project with the following environment variables:
+
+```
+NEXT_PUBLIC_API_BASE_URL=https://stag.api.storyprotocol.net
```
Comment (edisonz0718) [inline README.md:30]: Please remove the value for the API URL for now

File: README.md [hunk 2]
```diff
@@ -1,13 +1,140 @@
...
+
+// Instantiate the Story Client for readonly operations, using default 
+export const realonlyClient = StoryClient.newReadOnlyClient({});
+
+// Instantiate the Story Client, test environment required for alpha release.
+// The private key is also required for written operations.
+export const client = StoryClient.newClient({account});
+```
+
+Here we need access accounts through the package `viem` (https://www.npmjs.com/package/viem). The package should already been installed automatically with Story Protocol core SDK.
```
Comment (edisonz0718) [inline README.md:93]: I think we won't install the viem package for the users. Users still need to install for themselves. So the language here is a bit confusing. Maybe we can remove the latter sentence

File: README.md [hunk 3]
```diff
@@ -18,15 +145,15 @@ Under the `typescript-sdk/packages/core-sdk` directory:
 To set up your testing environment (e.g., a new Next.js project), use `yalc add @story-protocol/core-sdk@0.1.0-rc.7` (ensure the version number is updated accordingly).
```
Comment (edisonz0718) [inline README.md:145]: can we just use a placeholder like <version> to replace 0.1.0-rc.7

File: README.md [hunk 4]
```diff
@@ -1,32 +1,158 @@
...
+
+Here is the way to create a Story Protocol client with the private key:
+
+```typescript
+import { createWalletClient, custom } from 'viem'
+import { sepolia } from 'viem/chains'
+
+const walletClient = createWalletClient({
+  chain: sepolia,
+  transport: custom(window.ethereum)
```
[the remark this answers is not in the document]
Comment (z-samuel) [reply README.md:65]: I was referring to this section for the code: https://viem.sh/docs/clients/wallet.html#json-rpc-accounts , the `custom(window.ethereum)` is assigned to `transport` field in the doc.

**Review by edisonz0718 — APPROVED** [review]

**Pull request merged**